wcivf/apps/feedback/views.py

```diff
@@ -12,6 +14,20 @@ class FeedbackFormView(UpdateView):
     form_class = FeedbackForm
     template_name = "feedback/feedback_form_view.html"
 
+    @property
+    def is_spam(self):
+        akismet = Akismet(
+            settings.AKISMET_API_KEY, blog=settings.AKISMET_BLOG_URL
+        )
+
+        if not akismet.api_key:
```

Comment:
Not a big deal, but if the implementation of `Akismet` changed so that an error is thrown on init with a bad API key (like we had seen previously in CI) this could then cause a 500 error if we didn't have `settings.AKISMET_API_KEY` set. So while not a big deal, it would be safer to check the `settings.AKISMET_API_KEY` directly and return false before trying to initialise `Akismet` e.g:
```
@property
def is_spam(self):
    if not settings.AKISMET_API_KEY:
        return False

    akismet = Akismet(
        settings.AKISMET_API_KEY, blog=settings.AKISMET_BLOG_URL
    )
    return akismet.check(
        self.request.META["REMOTE_ADDR"],
        self.request.META["HTTP_USER_AGENT"],
        comment_content=self.request.POST.get("comments"),
    )



```